client/src/pages/HomePage.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from "react";
+import "bulma/css/bulma.min.css";
+import styles from "./HomePage.module.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+import { Link, useLocation } from "react-router-dom";
+import Header from "../components/Header";
+
+function HomePage() {
+  return (
+    <div>
+      <div className={styles.homePageContainer}>
+      <div className={styles.sections}>
+        <h4 className="title is-4 subtitle">Sections</h4>
+
```

Comment:
Please look at this commit and make sure it gets merged in
https://github.com/IJNA/the-orb/commit/45792281306f6a9a2cfe04fdd59bf91a863c1277
Thanks!

Reply:
@wyatthilde 
I made another minor UI update

Here is that commit: https://github.com/IJNA/the-orb/commit/25b0a80093f02d31da7ef0155b99733176fca370

Thanks!